src/yetibot/commands/github.clj

```diff
@@ -136,6 +137,50 @@
                   (f/unparse date-formatter datetime))))
       (str n " is not a number"))))
 
+(defn show-latest-release-info-cmd
+  "gh releases show <org>/<repo-name> # retrieve info about the latest release on a Github repository"
+  {:yb/cat #{:util :info}}
+  [{[_ org-name repo] :match}]
+  (let [latest-release (gh/latest-releases org-name repo)
+        status (:status latest-release)]
+    (if (nil? status)
```

Comment:
Let's use a helper function to reduce the similar code between `show-latest-release-info-cmd` and `show-release-info-by-tag-cmd`. Maybe call it `format-release`.

Reply:
I'm on it ^^